psutil/_psutil_osx.c

```diff
@@ -836,6 +836,51 @@ psutil_disk_partitions(PyObject *self, PyObject *args) {
 }
 
 
+static PyObject *
+psutil_disk_usage_used(PyObject *self, PyObject *args) {
+    PyObject *default_value;
+    PyObject *mount_point_bytes = NULL;
```

Comment:
another minor suggestion: there is a convention in psutil C code to use the `py_` prefix for the `PyObject` variables. Could you rename these 2 guys to `py_default_value` and `py_mount_point_bytes`?